branding/supplemental-ui/uyuni/uyuni-webui/css/site-extra.css

```diff
@@ -905,3 +905,30 @@ img.errorsvg {
 }
 
 }
+
+
+
+
+/* Language Selection */
+
+
+img.langIcon.english{
+
+  width: 27px;
+  margin-right: 6px;
+}
+
+img.langIcon.spanish{
+
+  width: 27px;
+  margin-right: 6px;
+}
+
+
+#en{
+  cursor: pointer;
+}
+
+#es{
+  cursor: pointer;
+}
```

Comment:
```suggestion

/* Language Selection */

img.langIcon.english {
  width: 27px;
  margin-right: 6px;
}

img.langIcon.spanish {
  width: 27px;
  margin-right: 6px;
}

#en {
  cursor: pointer;
}

#es {
  cursor: pointer;
}

```
This is just a general formatting suggestion, i.e. I would recommend to not put so many empty lines for better readability, use a blank before the `{`, etc. Please consider e.g. [these CSS formatting rules](https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules) for further general information.